src/locales/en.yml

```diff
@@ -24,15 +24,25 @@ commands:
       expect-image: Expect an image.
       invalid-input: Invalid prompt, only English words can be used.
       forbidden-word: Forbidden words in prompt.
-      concurrent-jobs: Please wait and retry later.
-      waiting: Request queued, please wait for generation
+      concurrent-jobs@random:
+        - Too busy to handle your request...
+        - "Brb power nap :zzz:"
+        - (*~*) Have no time to draw a new one.
+      waiting@random:
+        - The illustrator starts painting.
+        - Monet and Da Vinci, whose style is better for this?
+      pending@plural:
+        - Sure.
+        - Sure, but please wait for me to complete this one before.
+        - Bruh, there are {0} jobs pending!
       nickname: Novel AI
       invalid-size: The Enhance mode can only be used for images generated. Use /novelai without enhance option if you are using normal img2img.
       file-too-large: File is too large.
       unsupported-file-type: Unsupported file type.
       download-error: Parsing image failed.
       unknown-error: An unknown error occurred.
       response-error: An unknown error occurred ({0}).
+      empty-response: The server didn't return a valid image.
```

Comment:
same as https://github.com/koishijs/novelai-bot/pull/55#discussion_r998891515 , which should I write.